lib/elixir_search_extractor/search_keyword/queries/keyword_query.ex

```diff
@@ -10,4 +10,10 @@ defmodule ElixirSearchExtractor.SearchKeyword.Queries.KeywordQuery do
       where: file.user_id == ^user.id,
       order_by: fragment("? DESC", keyword.inserted_at)
   end
+
+  def filter_keywords(query, nil), do: query
```

Comment:
Let's add test for this function

Reply:
@hanam1ni added in [6028f82](https://github.com/rafayet-monon/elixir_search_extractor/pull/44/commits/6028f8286c4f53f05257447453568337ba88a59a)